cpp/CMakeLists.txt

```diff
@@ -788,11 +790,23 @@ message(STATUS "RMM_LOGGING_LEVEL = '${RMM_LOGGING_LEVEL}'.")
 
 target_compile_definitions(cudf PUBLIC SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_${RMM_LOGGING_LEVEL})
 
+###################################################################################################
+# - nvtx ------------------------------------------------------------------------------------------
+CPMAddPackage(
```

Comment:
I think this supports the GIT_SHALLOW option?

Reply:
done